build/config/android/config.gni

```diff
@@ -94,20 +94,19 @@ if (is_android) {
 
   # Subdirectories inside android_ndk_root that contain the sysroot for the
   # associated platform.
-  _android_api_level = 16
+  _android_api_level = 24
```

Comment:
We usually want to use the latest available NDK platform (barring issues with the toolchain, etc.) and then depending on the features we use, update the AndroidManifest to gate incompatible devices.